source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -0,0 +1,334 @@
+*   on VDSM, the VIF Driver, invoked using the VDSM before_nic_hotplug/before_device_create hook,
+    will connect the VM NIC to the network provided by the external provider by modifing the device xml
+
+![](external_network_provider_schema1.png "fig:external_network_provider_schema1.jpg")
+
+### Vnic unplug
+
+When a VNIC is unplugged, libvirt will automatically unplug the port from
+the OVS bridge.
+This is the equivalent of executing the following OVS commands:
+	ovs-vsctl del-port <nic name>
+
+This deletes the port from the OVN integration bridge. OVN controller 
+modifies the local OpenFlows on the host and deletes the chassis id from the
+bindings. Other OVN-controllers notice the binding change, and update their
+local flows.
+
+## Packaging and installation
+
+### OVN Central Server
+The OVN central server must be intalled manually. It must be accessible via network to
+the OVN provider and to all the hosts which use OVN.
+
+### OVN Controller
+The OVN Controller must be installed on each host using OVN.
+After installation the OVN controller must be configured with the value of the OVN central
+which it will be using and the local IP used for tunneling.
+This could be automated using an otopi plugin, but this would require a one-time configuration
+of the plugin. A way to establish the tunneling IP on the host should be provided, as well as
+the IP of the OVN central.
+ A look at the plugin directory might be helpful to understand the process:
+/usr/share/ovirt-host-deploy/plugins/ovirt-host-deploy
+
+Another possibility would be to to use vdsm-tool configure.
+
+### Provider packaging and installation
+The provider will be delivered as an RPM. After installation the only configurable
+property will be the IP of the OVN north DB. If not specified, the provider will assume
+it is running on the same host as the provider is installed on.
+
+### Driver packaging
+The VIF driver will probably be delivered as an rpm. As such, it will be installed manually
+on the hosts. The VIF driver connects to the local OVS instance and OVN north DB instance used
+by the local OVS. It needs no further configuration.
+
+The VIF driver RPM could be made available in the VDSM repo, or a local repo available to the hosts. 
+It could then be installed by being added to:
+/usr/share/ovirt-host-deploy/plugins/ovirt-host-deploy/vdsmhooks/packages.d\
+All RPM files specified there will be installed on the host during the host deploy operation.
+There should be a separate file for each linux distribution. The file extension
+must be the name of the linux distribution on which the rpm's are to be installed
+(fedora, redhat, ...).
+
+## Further considerations
+
+### IPAM
+The IP assigned to OVN managed NICs should be assigned from a subnet (pool of IPs) definded within OVN.
+As of now, this is still not implemented in OVN.
+A request for this functionality has been created in the OVS bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1368043
```

Comment:
please use nicer format to name urls. e.g.
in the [OVS bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1368043)